### ISSUE TITLE
Official Docker Image

### ISSUE THREAD

**Issue by ahmetegesel** [codelion/optillm#115]
Can you add pushing an official docker image to the release pipeline? It would be more convenient to install for those who want to run it on Docker

**Comment by ahmetegesel** [comment]
Hey, thank you very much for quick resolution. Can you also please update the installation guide? I could only find how to use it by checking PR attached to the issue

**Comment by codelion** [comment]
yes, I just added a note here - https://github.com/codelion/optillm?tab=readme-ov-file#using-docker
